default/python/AI/AIDependencies.py

```diff
@@ -399,6 +406,22 @@
     "SHP_SOLAR_CONT",
 )
 
+
+# Industry modifiers per population - [[EARLY_PRIORITY]]: Affected by species modifiers
+INDUSTRY_EFFECTS_PER_POP_MODIFIED_BY_SPECIES = {
```

Comment:
Sorry for tags, it is from other places.

Extract code with cycle `for tech in ind_tech_map_before_species_mod` to function.